nomos-da/kzgrs-backend/src/common/mod.rs

```diff
@@ -19,6 +19,7 @@ pub struct Row(pub Vec<Chunk>);
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Column(pub Vec<Chunk>);
 pub struct ChunksMatrix(pub Vec<Row>);
+pub type ColumnIndex = u32;
```

Comment:
Actually now that I think of this. We should probably use the smallest fitting size which would be `u16`?